Misc/NEWS.d/next/Library/2022-01-11-15-54-15.bpo-46333.B1faiF.rst

```diff
@@ -0,0 +1,4 @@
+The :meth:`__eq__` and :meth:`__hash__` methods of
+:class:`typing.ForwardRef` now honor the ``module`` parameter of
+:class:`typing.ForwardRef`. Forward references from different
+modules are now differentiated.
```

Comment:
The lack of newline at the end of the file is causing the docs-related tests to fail due to a new rst-linting script that was added to the CI a few days ago

```suggestion
modules are now differentiated.

```